DBZ-4332 Update documentation about `row_id` (#5527)
debezium/debezium: documentation/modules/ROOT/pages/connectors/oracle.adoc

|===

[[oracle-openlogreplicator-row-id-support]]
=== OpenLogReplicator ROWID support

OpenLogReplicator does not provide the `ROWID` pseudocolumn details in its change event payloads.

> Consequently, the {prodname} connector for Oracle will consistently display empty `row_id` values in the source information block when utilizing the OpenLogReplicator adapter.

Reviewer: ```suggestion
Consequently, in environments that use the OpenLogReplicator adapter, the {prodname} connector for Oracle consistently displays empty `row_id` values in the source information block.
```
